cfme/tests/configure/test_analysis_profiles.py

```diff
@@ -24,13 +24,107 @@
     {'Name': files_list[0]['Name'],
      'Collect Contents?': not files_list[0]['Collect Contents?']}]
 
+TENANT_NAME = "tenant_{}".format(fauxfactory.gen_alphanumeric())
+
+PRODUCT_FEATURES = [
+    [
+        (
+            [
+                "Everything",
+                "Settings",
+                "Configuration",
+                "Access Control",
+                "Tenants",
+                "Modify",
+                "Manage Quotas",
+                "Manage Quotas (My Company)",
+            ],
+            False,
```

Comment:
There appears to be an extra layer of lists in this structure, since there is only a single item in each. Do you expect further parametrization items, with multiple tuples?

Reply:
Removed tuples from parametrization.